lambdatimes/src/components/Content/Tabs.js

```diff
@@ -1,16 +1,26 @@
 import React from 'react';
 import Tab from './Tab';
+import PropTypes from 'prop-types';
+
+
+
 const Tabs = props => {
   return (
     <div className="tabs">
       <div className="topics">
         <span className="title">TRENDING TOPICS:</span>
-        {/* map over the tabs provided on your props, create a new Tab component for each one.
-            give the tab component a `selectTabHandler`, the `selectedTab`, and the `tab` itself as props*/}
+        {props.tabs.map((x, y)  => (
+            <Tab key={y} tab={x} selectTabHandler={props.changeSelected} selectedTab={props.selectedTab} />
```

Comment:
Using the index from the map method as a key is never a good idea. In this instance it works because the tabs never change. But, if tabs changed based on the data it receives, you would get unwanted behavior out of React here